include/perf_watcher.h

```diff
@@ -9,8 +9,14 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+typedef enum {
+  PWRequestedFalse = 0, // always off
+  PWRequestedTrue,      // always on
+  PWPreferedTrue,       // On if possible, default to OFF on failure
+} PerfWatcherValue;
```

Comment:
The naming here is not great. C++ scopes might help. 
perf_watcher::Options::Values

Ideas are welcome